src/translations/nl.json

```diff
@@ -0,0 +1,219 @@
+{
```

Comment:
As Dutch is not being released as part of this PR, it should also not be added here.